director/director.go

```diff
@@ -105,6 +105,10 @@ var (
 	statUtilsMutex = sync.RWMutex{}
 )
 
+func init() {
+	HookServerAdsCache()
```

Comment:
I think this is the last change request from me -- now that this isn't being access by another package, there's no need to export. Can you lowercase the function name?